### DISCUSSION TITLE
Options

### DISCUSSION THREAD

**Discussion by Rodeo313** [opynfinance/OGP#16] [Ideas]
Hi,

Is it possible to build straddles and strangles on Ether options

**Comment by southamerica** [comment]
> Hi,
> 
> Is it possible to build straddles and strangles on Ether options

Hey! Yup, Ribbon is an example of a protocol that launched on-chain ETH Strangle options using Opyn options!

**Reply by Lucas-Kohorst** [reply]
The strangles app that was available [here](https://strangles.ribbon.finance/) is deprecated as of June 15th.
